package.json

```diff
@@ -137,14 +137,14 @@
     "mydatepicker": "2.1.0",
     "ng2-completer": "1.6.3",
     "ng2-dragula": "1.5.0",
-    "ng2-restangular": "0.1.30",
     "ng2-truncate": "1.3.11",
     "ngx-base": "2.3.1",
```

Comment:
don't you need to update all the libs that you made changes to?

Reply:
Yup. I didn't realize that all those recent upversion PRs were been deleted instead of merged.